app/controllers/api_keys_controller.rb

```diff
@@ -0,0 +1,39 @@
+class ApiKeysController < ApplicationController
+  include ApiKeyable
+  before_action :redirect_to_signin, unless: :signed_in?
+
+  def index
+    @api_keys = current_user.api_keys
+    redirect_to new_profile_api_key_path if @api_keys.empty?
+  end
+
+  def new
+    @api_key = current_user.api_keys.build
+  end
+
+  def create
+    key = rubygems_key
```

Comment:
Consider tweaking `rubygems_key` name so it includes a verb that makes it clear this method generates an API key.